package.json

```diff
@@ -46,8 +46,9 @@
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "lint": "yarn run eslint; yarn run eslint-ts",
-    "eslint": "eslint --no-eslintrc  -c .eslintrc.js 'src/**'",
-    "eslint-ts": "eslint --no-eslintrc -c .eslintrc.json './src/**/*.ts*'",
+    "lint:fix": "yarn run eslint --fix; yarn run eslint-ts --fix",
```

Comment:
Use this cmd before you push or commit. It'll automatically fix some fixable errors. 

Also, we can apply [husky pre-commit](https://github.com/typicode/husky#usage) as well later if we're all good with :)